src/traces/indicator/plot.js

```diff
@@ -0,0 +1,788 @@
+    if(trace._hasNumber) data.push('number');
+    if(trace._hasDelta) {
+        data.push('delta');
+        if(trace.delta.position === 'left') data.reverse();
+    }
+    var sel = numbers.selectAll('tspan').data(data);
+    sel.enter().append('tspan');
+    sel
+        .attr('text-anchor', function() {return numbersAnchor;})
+        .attr('class', function(d) { return d;})
+        .attr('dx', function(d, i) {
+            // Add padding to the second tspan when it's a one-liner
+            if(i === 1) {
+                var pos = trace.delta.position;
+                if(pos === 'left' || pos === 'right') return 10;
+            }
+            return null;
+        });
+    sel.exit().remove();
+
+    function drawBignumber() {
+        // bignumber
+        var bignumberAx = mockAxis(gd, {tickformat: trace.number.valueformat});
+        var fmt = function(v) { return Axes.tickText(bignumberAx, v).text;};
+        var bignumberSuffix = trace.number.suffix;
+        if(bignumberSuffix) bignumberSuffix = ' ' + bignumberSuffix;
+
+        var number = numbers.select('tspan.number');
+        number
+            .call(Drawing.font, trace.number.font)
+            .attr('x', null)
+            .attr('dy', bignumberY);
+
+        if(hasTransition) {
+            number
+                .transition()
+                .duration(transitionOpts.duration)
+                .ease(transitionOpts.easing)
+                .each('end', function() { trace._lastValue = cd[0].y; onComplete && onComplete(); })
+                .each('interrupt', function() { onComplete && onComplete(); })
+                .attrTween('text', function() {
+                    var that = d3.select(this);
+                    var interpolator = d3.interpolateNumber(cd[0].lastY, cd[0].y);
+                    return function(t) {
+                        that.text(fmt(interpolator(t)) + bignumberSuffix);
+                    };
+                });
+        } else {
+            number.text(fmt(cd[0].y) + bignumberSuffix);
+        }
+    }
+
+    function drawDelta() {
+        // delta
+        var deltaAx = mockAxis(gd, {tickformat: trace.delta.valueformat});
+        var deltaFmt = function(v) { return Axes.tickText(deltaAx, v).text;};
+        if(!trace._deltaLastValue) trace._deltaLastValue = 0;
+        var deltaValue = function(d) {
+            var value = trace.delta.relative ? d.relativeDelta : d.delta;
+            return value;
+        };
+        var deltaFormatText = function(value) {
+            if(value === 0) return '-';
+            return (value > 0 ? trace.delta.increasing.symbol : trace.delta.decreasing.symbol) + deltaFmt(value);
+        };
+        var deltaFill = function(d) {
+            return d.delta >= 0 ? trace.delta.increasing.color : trace.delta.decreasing.color;
+        };
+        var delta = numbers.select('tspan.delta');
+        delta
+            .call(Drawing.font, trace.delta.font)
+            .style('fill', deltaFill)
```

Comment:
Please use `Color.fill`:

https://github.com/plotly/plotly.js/blob/795220f815afcf79c73885715379b606afc1b4a1/src/components/color/index.js#L89-L95

... for similar reasons as `Color.stroke`.

Reply:
Done in e6d8525ab07c56ca10bdbac6762b06d1d57bf830